fe/be-java-extensions/paimon-scanner/src/main/java/org/apache/doris/paimon/PaimonColumnValue.java

```diff
@@ -32,12 +35,18 @@
 
 public class PaimonColumnValue implements ColumnValue {
     private int idx;
-    private InternalRow record;
-    ColumnType dorisType;
+    private DataGetters record;
```

Comment:
why change `InternalRow` to `DataGetters`

Reply:
When dealing with `unpackArray` and `unpackMap`, since `org.apache.paimon.data.InternalArray` extends `org.apache.paimon.data.DataGetters`, if do not change the `InternalRow` to `DataGetters`, the force transfer will fail.